app/models/alchemy/element.rb

```diff
@@ -284,12 +284,12 @@ def to_partial_path
     #
     # If the page is the current preview it uses the element's updated_at value as cache key.
     #
+    # Larry's Note:
+    # The following method has been monkey-patched to never use 'published_at' and always use 'updated_at'
```

Comment:
Metrics/LineLength: Line is too long. [106/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)